_posts/2020-11-23-micro-grants-in-2021-ols-3-ols-4.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: OLS Community support micro grants
+authors:
+- bebatut
+- malvikasharan
+- yochannah
+image: https://images.unsplash.com/photo-1579621970563-ebec7560ff3e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1951&q=80
+photos:
+  name: Photo by Micheile Henderson on Unsplash
+  url: https://unsplash.com/@micheile
+---
+
+Open Life Science has recently been lucky enough to be awarded two small grants - one from the EOSC-Life project, which will partially support OLS-3, and two more small grants which will partially support OLS-3 and 4. In all cases, the funds awarded are less than the full cost of a cohort if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a helpful start, and one we are privileged to have.
+
+For OLS-3 and OLS-4 we are delighted to announce that all project leaders (aka mentees) - will be able to request small seed grants for expenses that they may need for their participation in OLS or to run their project. This could be an audio headset, the cost for more data on a tethered mobile phone to attend calls, fliers or stickers with your project logo on it, or the fee to register your project as a legal organisation in your country.
```

Comment:
```suggestion
With this development, we are delighted to announce that all OLS-3 and OLS-4 project leaders (aka mentees) - will be able to request small seed grants for expenses that they may need for their participation in OLS or to run their project. This could be an audio headset, the cost for more data on a tethered mobile phone to attend calls, fliers or stickers with your project logo on it, or the fee to register your project as a legal organisation in your country.
```